layouts/_default/baseof.html

```diff
@@ -23,6 +23,8 @@
     {{ if .Site.Params.cookie_consent_info_url }}
     {{ partial "cookie-consent.html" . }}
     {{ end }}
+
+		{{ partial "head-custom.html" . }}
```

Comment:
Let's stick to one logical change for each commit / pull request — you can add modelines to ALL the files in a separate pull request. ;)

Also, in the future it would be better for the GitHub workflow if you committed your changes to a feature branch like `custom-head-code` or something. Your `master` should theoretically always match the upstream one (mine).